fc/fullcontact.go

```diff
@@ -276,8 +282,12 @@ func (fcClient *fullContactClient) IdentityDelete(resolveRequest *ResolveRequest
 }
 
 func (fcClient *fullContactClient) resolveRequest(ch chan *APIResponse, resolveRequest *ResolveRequest, url string) chan *APIResponse {
+	err := validateResolveRequest(resolveRequest)
```

Comment:
We dont need this validation for `resolve` and `delete`, so we can just call this inside `validateForIdentityMap` in `resolve_request.go`

Reply:
I didn't get it. I guess if we dont use the `validateResolveRequest` here, then there is no reference to this validation at all anywhere. So should we remove the `validateResolveRequest` section itself?

Reply:
We can call it inside https://github.com/fullcontact/fullcontact-go/blob/master/fc/resolve_request.go#L57
as we need to only validate mfReq in `identity.map`